AUTHORS.md

```diff
@@ -18,17 +18,17 @@ the following authors:
         </a>
     </td>
     <td align="center">
-        <a href="https://github.com/Float07">
-            <img src="https://avatars.githubusercontent.com/u/39651883?v=4" width="100;" alt="Float07"/>
+        <a href="https://github.com/Dauboau">
+            <img src="https://avatars.githubusercontent.com/u/86164187?v=4" width="100;" alt="Dauboau"/>
             <br />
-            <sub><b>Thiago Daniel Cagnoni De Pauli</b></sub>
+            <sub><b>Daniel Contente Romanzini</b></sub>
         </a>
     </td>
     <td align="center">
-        <a href="https://github.com/Dauboau">
-            <img src="https://avatars.githubusercontent.com/u/86164187?v=4" width="100;" alt="Dauboau"/>
+        <a href="https://github.com/Float07">
+            <img src="https://avatars.githubusercontent.com/u/39651883?v=4" width="100;" alt="Float07"/>
             <br />
-            <sub><b>Daniel Contente Romanzini</b></sub>
+            <sub><b>Thiago Daniel Cagnoni De Pauli</b></sub>
         </a>
     </td>
     <td align="center">
```

Comment:
It's better to not manually edit this file for we have a automated workflow to update it since #37

Reply:
It was a github action, not mine...

Reply:
Any clue in how to remove this?

Reply:
I guess it just changed the order of the profiles. As it's an github automatic action, perhaps it isn't that bad.